conda_forge_tick/update_upstream_versions.py

```diff
@@ -23,6 +24,17 @@
 logger = logging.getLogger("conda-forge-tick._update_versions")
 
 
+def verify(version: str, next_version: str):
```

Comment:
This function needs some unit testing.

Reply:
inside it? I was planning to create a test script for the profiler and use the opportunity to create one for this one too.

Reply:
no i mean in the tests dir - it is supposed to order versions - we should make sure it does so - you might also consider using the Version class from conda itself

Reply:
I will have a look at the Version class, and think os a test for this too (maybe using the test information of `update_upstream_versions`)

Reply:
I saw one of the discussions about the Verison ordering class of Conda, and I learned a lot with that hshshs, I will try to use theirs. As a test code, I will use the current `test_update_upstream_versions` with some modifications...